### ISSUE TITLE
Monika Full Week Mod?

### ISSUE THREAD

**Issue by OwOCakes** [luckydog7/Funkin-android#1243]
https://github.com/aetherdx/monika

Source code for the full week

**Comment by JoseGithubFNF** [comment]
> https://github.com/aetherdx/monika
> 
> Source code for the full week

Wait For Gamerbross Port Idk?

**Comment by OwOCakes** [comment]
its the one that recently came out

**Comment by OwOCakes** [comment]
https://m.youtube.com/watch?v=BLCY_x9vTzU

**Comment by JoseGithubFNF** [comment]
> https://m.youtube.com/watch?v=BLCY_x9vTzU
> The vid of the mod

Yes I Mean That Mod But I Would Consider Waiting For Gamerbross Port Or Just Wait For someone To Port If Or You Just Port Yourself idk

**Comment by Tonate12** [comment]
Here https://www.mediafire.com/file/7a9in4wzmzsqsr9/Monika-deb
